src/widgets/panel/__tests__/panel-test.ts

```diff
@@ -25,7 +25,7 @@ beforeEach(() => {
 describe('Usage', () => {
   test('without arguments does not throw', () => {
     expect(() => {
-      panel();
+      panel({});
```

Comment:
this test is to see that we don't destructure without default, giving an object changes it :)

Reply:
Fixed, thanks.